#手柄面板正面右侧4个按钮中的上下两个(Y A 或者 1 3)，用作测试用途时的分配，更改后重启joy手柄节点即可，无需编译
#0-不使用测试功能， 1-用作开、关超声波避障，2-用作收、缩充电杆，
#3-用作开、关防跌落功能，4-同时用作开、关（超声波、防跌落）
useTestBtn: 5

#导航记录起始点
node_start: 0
#导航点累计间隔
node_increment: 2
dev: "/dev/input/js0"
autorepeat_rate: 10
#手柄速度设置  m/s
walk_vel: 0.2
#run_vel: 0.6  #60cm/s  1.0=100cm/s, (武汉验收)加速后制动测试时使用大速度，转弯半径测试时如果是手柄控制，采用小速度
run_vel: 0.6
walk_vel_y: 0.2
run_vel_y: 0.4
yaw_rate: 0.5
yaw_rate_run: 0.8

pantilt_increment: 5.0

ev_increment: 0.03

#按键码设置
KEY_FORWARD_BACKWARD: 7

KEY_LEFT_RIGHT: 6

KEY_TURN_LEFT: 2

KEY_TURN_RIGHT: 1


KEY_SPEED_UP: 2

KEY_PAUSE: 5

KEY_OBSTACLE: 4

KEY_POSE_WRITE: 5

#KEY_PANTILT_ON: 4

#KEY_PAN_LEFT: 0

#KEY_PAN_RIGHT: 2

#KEY_TILT_UP: 3

#KEY_TILT_DOWN: 1

#KEY_EV_UP: 5


